README.md

```diff
@@ -14,10 +14,9 @@ UCX will guide you, the Databricks customer, through the process of upgrading yo
 1. The upgrade process will first install code, libraries, and workflows into your workspace.
 2. After installation, you will run a series of workflows and examine the output.
 
-UCX leverages Databricks Lakehouse platform to upgrade itself. The upgrade process includes creating jobs, notebooks, and deploying code and configuration files. The `install.sh` guides you through the installation.
-
-By running the installation you install the assessment job and several upgrade jobs. The assessment and upgrade jobs are outlined in the custom-generated README.py that is created by the installer and displayed to you by the `install.sh`. See interactive installation [tutorial here](https://app.getreprise.com/launch/G6YbWLn/).
+UCX leverages Databricks Lakehouse platform to upgrade itself. The upgrade process includes creating jobs, notebooks, and deploying code and configuration files. 
```

Comment:
Does it upgrade itself? :)

Reply:
I think the install.sh is still useful for development to run manual tests from your local codebase. It would be great if we could put it under contribution somewhere.

Reply:
`databricks labs install .` for dev mode